clang/lib/CIR/CodeGen/CIRGenExprConst.cpp

```diff
@@ -1050,16 +1050,71 @@ class ConstantLValueEmitter
 
   bool hasNonZeroOffset() const { return !Value.getLValueOffset().isZero(); }
 
-  /// Return the value offset.
-  mlir::Attribute getOffset() { llvm_unreachable("NYI"); }
-
   // TODO(cir): create a proper interface to absctract CIR constant values.
 
+  // Convert byte offset to sequence of high-level indices...
+  void computeIndices(int64_t Offset, mlir::Type Ty, CIRDataLayout Layout,
```

Comment:
This probably should be a common helper since it will likely be used in more places, living somewhere under `CIR/Dialect`, maybe name it `computeGlobalViewIndicesFromFlatOffset`? Also add some comments explaining that this isn't ideal but that we don't wanna get intrusive to other parts in Clang for the moment.

Reply:
Done.